driver-ui/src/main/java/eu/tsystems/mms/tic/testframework/webdrivermanager/WebDriverCapabilityLogHelper.java

```diff
@@ -71,12 +71,14 @@ private Map<String, Object> shortChromeExtensionStrings(Map<String, Object> capa
         if (chromeOptionsObject != null) {
             final Map chromeOptions = (Map) chromeOptionsObject;
             if (chromeOptions.containsKey("extensions")) {
-                chromeOptions.put("extensions", shortAllStringsInLists(chromeOptions.get("extensions")));
+                List<String> extensions = shortAllStringsInLists(chromeOptions.get("extensions"));
+                if (extensions.size() > 0) chromeOptions.put("extensions", extensions);
             }
         }
 
         if (extensionsObject != null) {
-            capabilityMap.put("extensions", shortAllStringsInLists(extensionsObject));
+            List<String> extensions = shortAllStringsInLists(extensionsObject);
+            if (extensions.size() > 0) capabilityMap.put("extensions", shortAllStringsInLists(extensionsObject));
```

Comment:
Here the change _does_ affect behaviour, because `capabilityMap` is a fresh, initially empty map.
(And FWIW I consider the change sensible. Gotta hate empty map entries.)